gapic/lang/common.yaml

```diff
@@ -5,6 +5,7 @@ common:
   package_dependencies_yaml: ${REPOROOT}/googleapis/gapic/packaging/dependencies.yaml
   package_defaults_yaml: ${REPOROOT}/googleapis/gapic/packaging/api_defaults.yaml
   common_protos_yaml: ${REPOROOT}/googleapis/gapic/packaging/common_protos.yaml
+  organization_name: google-cloud
```

Comment:
Actually, I realize this doesn't work; according to the current recommended artman `USAGE`, the values in common.yaml override the values in artman_{api}.yaml.